guides/manual-installation.md

```diff
@@ -0,0 +1,505 @@
+## Scope
+
+This documents covers the step to manually install trento server without relying on the trento helm chart or kubernetes all together. The lastest
+available version of SUSE Linux Enterprise Server for SAP Applications is used as the base operating system, which is 15 SP5 at the time of writing.
+
+For other installation options, check:
+
+- https://github.com/trento-project/helm-charts/
+- https://github.com/trento-project/ansible (WIP)
+
+## List of dependencies
+
+- prometheus (optional)
+- postgresql
+- rabbitmq
+- docker runtime
+
+## Installation
+
+### Install prometheus (Optional)
+
+[Prometheus](https://prometheus.io/) allows trento to collect metrics from the monitored hosts. It is not required to run trento, but it is
+recommended. At the time of this writing, there is no officially supported Prometheus package in SLES 15 SP5. However, there are a few options to
+install it:
+
+#### Option 1: Use existing installation
+
+If you already have a prometheus server running, you can reuse your existing installation. You will need to provide the URL to the prometheus server
+in the `PROMETHEUS_URL` environment variable when running the trento-web container.
+
+#### Option 2: Install prometheus using the **unsupported** PackageHub repository
+
+> Note: PackageHub packages are tested by SUSE, but they do not come with the same level of support as the core SLES packages.
+> Users should assess the suitability of these packages based on their own risk tolerance and support needs.
+
+Enable PackageHub repository:
+
+```bash
+SUSEConnect --product PackageHub/15.5/x86_64
+```
+
+Add the prometheus user/group:
+
+```bash
+groupadd --system prometheus
+useradd -s /sbin/nologin --system -g prometheus prometheus
+```
+
+After a successful registration, you can install prometheus using zypper:
+
+```bash
+zypper in golang-github-prometheus-prometheus
+```
+
+We will be prompted that a missing dependency can't be satisfied:
+
+```bash
+Problem: nothing provides 'group(prometheus)' needed by the to be installed golang-github-prometheus-prometheus-2.37.6-150100.4.17.1.x86_64
+ Solution 1: do not install golang-github-prometheus-prometheus-2.37.6-150100.4.17.1.x86_64
+ Solution 2: break golang-github-prometheus-prometheus-2.37.6-150100.4.17.1.x86_64 by ignoring some of its dependencies
+```
+
+As we have added the prometheus user/group, we can safely ignore this warning and proceed with the installation (option 2).
+
+Enable and start the prometheus service:
+
+```bash
+systemctl enable --now prometheus
+```
+
+Allow prometheus to be accessible from docker. Add an exception on firewalld:
+
+```bash
+firewall-cmd --zone=public --add-port=9090/tcp --permanent
+firewall-cmd --reload
+```
+
+### Install postgresql
+
+```bash
+zypper in postgresql-server
+```
+
+Enable and start postgresql:
+
+```bash
+systemctl enable --now postgresql
+```
+
+#### Configure postgresql
+
+**Step 1:** Start `psql` to open a connection to the database:
```

Comment:
"with a user with the correct privileges to run the commands", idk how to phrase it